core/app/assets/stylesheets/refinery/sections/_menu_item_icons.scss

```diff
@@ -0,0 +1,9 @@
+.site-name {@include icon('bars', $plugin_icon_colour, $margin: 1.2em)}
+
+.refinery_images_icon {@include icon('photo', $plugin_icon_colour, $margin: 1.2em)}
```

Comment:
Sorry Anita, I've been unable to do a thourough review of this PR today, but would it be helpful for us all to understand what problem we're trying to solve?

    <a href="/refinery/pages" class="plugin bump-down refinery_pages_icon">Pages</a>
    <a href="/refinery/pages" class="plugin bump-down files_icon">Pages</a>
    <a href="/refinery/pages"><i class="icon icon-files"></i> Pages</a>

What is the problem with the 3rd option there (the current html)?